cfg/TebLocalPlannerReconfigure.cfg

```diff
@@ -334,8 +334,8 @@ grp_optimization.add("obstacle_cost_exponent", double_t, 0,
 # Optimization/Omni
 grp_optimization_omni = grp_optimization.add_group("OptOmni", type="hide")
 
-grp_optimization_omni.add("norm_vel_lin", int_t, 0,
-	"Norm constraint for translational velocity (1: L1, 2: L2). For example, enforcing hypot(vx, xy) <= v_max_lin for L2",
+grp_optimization_omni.add("norm_vel_trans", int_t, 0,
```

Comment:
Q: what's the point of this?  what's the difference between enforcing L1 or L2?
To me u always want to enforce that the magnitude of vector vx, vy is <= max_vel_trans

Reply:
See https://github.com/rst-tu-dortmund/teb_local_planner/pull/346#discussion_r782823162

> It [L1] is way cheaper, and the behaviour is the one expected.
Sometimes optimization cannot keep up with L2 and the behaviour is not the one expected.